libs/configurable-product/driver/magento/src/product-configurable-driver.module.ts

```diff
@@ -25,8 +25,16 @@ export class DaffConfigurableProductMagentoDriverModule {
     return {
       ngModule: DaffConfigurableProductMagentoDriverModule,
       providers: [
-        ...daffProvideProductMagentoExtraProductFragments(magentoConfigurableProductFragment),
-        ...daffProvideProductMagentoExtraProductPreviewTransforms(transformMagentoConfigurableProduct),
+        {
+          provide: DAFF_PRODUCT_MAGENTO_EXTRA_PRODUCT_FRAGMENTS,
+          useValue: magentoConfigurableProductFragment,
+          multi: true,
+        },
+        {
+          provide: DAFF_PRODUCT_MAGENTO_PRODUCT_PREVIEW_TRANSFORM,
```

Comment:
I think this is the wrong token. You should be providing to the extra token: `DAFF_PRODUCT_MAGENTO_EXTRA_PRODUCT_PREVIEW_TRANSFORMS`

Reply:
oops, nice catch. I also switched out the fragments token for the `DAFF_PRODUCT_MAGENTO_EXTRA_PRODUCT_PREVIEW_FRAGMENTS`